### PR TITLE
Updating XM Cloud FAQ development section around migration-related questions

### LINKED ISSUE #681: XMC FAQ: Update content migration options
**URL:** https://developers.sitecore.com/learn/faq/xm-cloud/development#what-options-are-there-for-content-migration-from-earlier-sitecore-versions

1. Add link to new XMC migration tool download: https://developers.sitecore.com/downloads/xm-cloud#xm-to-xm-cloud-content-migration-tool
2. Add link to Razl support doc: https://support.sitecore.com/kb?id=kb_article_view&sysparm_article=KB1002803

### PR DESCRIPTION
**Pull request by jst-cyr**
## Description / Motivation

Update to the FAQ to remove 2022 SYM reference and direct users to more current migration options.

Fixes #681

## How Has This Been Tested?

Local and on Vercel preview

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation update (non-breaking change; modified files are limited to the `/data` directory or other markdown files)

## Checklist:
- [X] I have read the Contributing guide.
- [X] My code/comments/docs fully adhere to the Code of Conduct.
- [ ] My change is a code change.
- [X] My change is a documentation change and there are NO other updates required.
- [ ] My change has new or updated images which are stored in the `/public/images` folder that need to be migrated to Sitecore DAM

### CODE REVIEW

File: apps/devportal/data/markdown/pages/learn/faq/xm-cloud/development.md [hunk 1]
```diff
@@ -67,11 +67,31 @@ Sitecore also recommends as best practices:
...
+
+1. [Migration Advisor](https://migration.sitecore.com)
+
+   This application asks you questions about your current platform implementation and then suggests possible matching composable products to replicate your current functionality. It also recommends migration content you may be interested in based on your selections.
+
+1. [Composable DXP Migration content](/learn/getting-started/migrating-to-the-sitecore-composable-dxp)
+
+   This page on the Developer Portal lists out content related to migrating from the platform to the Composable DXP, including XM Cloud.
+
+1. [XM to XM Cloud content migration tool](/downloads/xm-cloud#xm-to-xm-cloud-content-migration-tool)
```
Comment (thomas-desmond) [inline apps/devportal/data/markdown/pages/learn/faq/xm-cloud/development.md:82]: All numbers in code are "1" but on page they are showing properly as 1, 2, 3. Can code be updated to 1, 2, 3.

File: apps/devportal/data/markdown/pages/learn/faq/xm-cloud/development.md [hunk 2]
```diff
@@ -67,11 +67,31 @@ Sitecore also recommends as best practices:
...
 ## What options are there for content migration from earlier Sitecore versions?
 
-Sitecore Content Serialization can be used for the migration of content. Sitecore is also looking to support Razl for migrating content from earlier versions.
+There are multiple options that can be used for content migration:
+
+1. Sitecore Content Serialization (SCS)
+
+1. [XM to XM Cloud content migration tool](/downloads/xm-cloud#xm-to-xm-cloud-content-migration-tool)
+
+1. [Razl](https://support.sitecore.com/kb?id=kb_article_view&sysparm_article=KB1002803)
```
Comment (thomas-desmond) [inline apps/devportal/data/markdown/pages/learn/faq/xm-cloud/development.md:94]: All numbers in code are "1" but on page they are showing properly as 1, 2, 3. Can code be updated to 1, 2, 3.

**Review by thomas-desmond — CHANGES_REQUESTED** [review]
I don't see how this is related to your code but the Table Of Contents does not update properly when you navigate to a new section of the FAQ. It's currently working in production. Is your branch behind `main`? 

image

**Comment by jst-cyr** [comment]
> I don't see how this is related to your code but the Table Of Contents does not update properly when you navigate to a new section of the FAQ. It's currently working in production. Is your branch behind `main`?

@thomas-desmond : That was really odd! I updated from Main yesterday, but it looks like the fix in Main happened after that. I just updated the branch with the latest from Main and sure enough the issue resolved itself locally. I've pushed that up to the branch and it should be available for review soon.

**Review by thomas-desmond — APPROVED** [review]
Looks good now!

**Pull request merged**